solidity/test/isms/ExternalBridgeTest.sol

```diff
@@ -115,7 +115,6 @@ abstract contract ExternalBridgeTest is Test {
     function test_postDispatch_revertWhen_insufficientValue() public {
         bytes memory encodedHookData = _encodeHookData(messageId, 0);
         originMailbox.updateLatestDispatchedId(messageId);
-        _expectOriginExternalBridgeCall(encodedHookData);
```

Comment:
what changed here?

Reply:
It throws an 'OutOfFunds` error before it gets a chance to call the externall bridge.

Reply:
fixed with removal